build/Dockerfile

```diff
@@ -128,7 +128,7 @@ RUN --mount=type=bind,from=alpine-fips-3.19,target=/tmp/fips/ \
 
 
 ############################################# Base image for Alpine with NGINX Plus, App Protect WAF and FIPS #############################################
-FROM alpine:3.17@sha256:53cf9478b76f4c8fae126acbdfb79bed6e69e628faff572ebe4a029d3d247d98 as alpine-plus-nap-fips
+FROM alpine:3.20@sha256:77726ef6b57ddf65bb551896826ec38bc3e53f75cdde31354fbffb4f25238ebd as alpine-plus-nap-fips
```

Comment:
@pdabelf5 is 3.20 fips certified?

Reply:
we build the fips module for specific alpine OS's and then import it 
```
RUN --mount=type=bind,from=alpine-fips-3.19,target=/tmp/fips/
```
so no, this will not currently work.